components/apimgt/org.wso2.carbon.apimgt.rest.api.admin.v1/src/main/resources/admin-api.yaml

```diff
@@ -1678,6 +1678,10 @@ paths:
         required: true
         schema:
           type: string
+      - name: organizationId
```

Comment:
@malinthaprasan shall we change this to organization and use existing xwso2TenantHeader

Reply:
As discussed offline, added a change to pick the organization from `RestApiUtil.getValidatedOrganization` method with commit: https://github.com/wso2/carbon-apimgt/pull/10807/commits/6b960157763fcfd0cd0e0ded0c26a1aee6ed9f19